pkg/discovery/discovery.go

```diff
@@ -226,3 +235,99 @@ func (d *tidbDiscovery) DiscoverDM(advertisePeerUrl string) (string, error) {
 	delete(currentCluster.peers, podName)
 	return fmt.Sprintf("--join=%s", strings.Join(mastersArr, ",")), nil
 }
+
+func (d *tidbDiscovery) VerifyPDEndpoint(pdURL string) (string, error) {
```

Comment:
The current logic only returns one PD address, which is vulnerable to failure, what about the following logic:

1. Define a slice for PD URL
2. No need to check the health of pdURL, append it to the slice.
3. Get TC from the pdURL
4. Go through the peerMembers, if they are healthy from `member.Health`(no need to call pd client anymore), append to the slice
5. Return all the URLs in the slice

WDYT @handlerww @csuzhangxc

Reply:
LGTM. a slice for PD URL should be better.

Reply:
OK, I will implement it.